Makefiles do not install test deps completely

## 🐞 Bug report
<!--
Thank you for reporting an issue in our project 🙌

Before opening a new issue, please make sure that we do not have any duplicates already open. You can ensure this by searching the issue list for this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead.
-->

### Describe the bug
<!-- A clear and concise description of what the bug is. -->
Running `make test` or `make integration-tests` in a hook folder does not install npm depencies from `/tests/integration`. The CI does this step manually.
### Steps To Reproduce
```
git clone https://github.com/secureCodeBox/secureCodeBox.git
cd hooks/cascading-scans
make test
```


### Expected behavior
<!-- A clear and concise description of what you expected to happen. -->
Integration tests should work


### Screenshots / Logs
<!-- If applicable, add screenshots to help explain your problem. -->

```
> @securecodebox/hook-cascading-scans@1.0.0 build
> npx tsc hook.ts --sourceMap --esModuleInterop

 FAIL  integration-tests/cascade-nmap-ncrack.test.js
  ● Test suite failed to run

    Cannot find module '@kubernetes/client-node' from '../../../tests/integration/helpers.js'

    Require stack:
      /tmp/secureCodeBox/tests/integration/helpers.js
      integration-tests/cascade-nmap-ncrack.test.js

      3 | // SPDX-License-Identifier: Apache-2.0
      4 |
    > 5 | const k8s = require("@kubernetes/client-node");
        |             ^
      6 |
      7 | const kc = new k8s.KubeConfig();
      8 | kc.loadFromDefault();

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (../../../tests/integration/helpers.js:5:13)

```
### Fix
<!-- Add any other context about the problem here. -->
Run `npm ci` in `/tests/integration`

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -558,10 +558,6 @@ jobs:
         working-directory: ./hooks
         run: npm ci
 
-      - name: "Install Test Dependencies"
-        working-directory: ./tests/integration
-        run: npm ci
-
       # ---- Unit-Test ----
 
       - name: "Run Unit Tests"

--- a/env-paths.mk
+++ b/env-paths.mk
@@ -13,6 +13,7 @@ HOOK_SDK_DIR				= $(PROJECT_DIR)/hook-sdk/nodejs
 AUTO_DISCOVERY_DIR			= $(PROJECT_DIR)/auto-discovery
 HELM_DOCS_DIR				= $(PROJECT_DIR)/.helm-docs
 TEMPLATES_DIR				= $(PROJECT_DIR)/.templates
+TESTS_HELPERS_DIR			= $(PROJECT_DIR)/tests/integration
 
 SCANNERS_CHART_LIST			:= $(sort $(wildcard $(SCANNERS_DIR)/*/Chart.yaml))
 SCANNERS_TEST_LIST			:= $(sort $(wildcard $(SCANNERS_DIR)/*/Makefile))

--- a/hooks.mk
+++ b/hooks.mk
@@ -54,4 +54,5 @@ deploy: ## 💾 Deploy this module via HelmChart into namespace "integration-tes
 integration-tests: ## 🩺 Start integration test for this module in the namespace "integration-tests"
 	@echo ".: 🩺 Starting integration test in kind namespace 'integration-tests'."
 	kubectl -n integration-tests delete scans --all
+	npm ci --prefix $(TESTS_HELPERS_DIR)
 	cd $(hook-prefix) && npm ci && npm run test --package jest@$(JEST_VERSION)

